test/users/npadmana/blas/c2chapel.py

```diff
@@ -0,0 +1,99 @@
+""" A C to Chapel parser to help automate writing bindings.
```

Comment:
Excellent - can you remove this script from the pull request, or does the module depend on it in some way?